models.py

```diff
@@ -179,21 +179,29 @@ def glimpse_network(images, locs, scope='glimpse_network'):
             # TODO: Lewis implements Glimpse Network here, using also
             # params['g_size'] and params['l_size']
             hidden_sensor_layer = tf.layers.dense(
-                    # not sure this is the correct input
                     inputs = patches,
                     units = params['g_size'],
                     activation = tf.nn.relu)
             
+            dense_sensor_layer = tf.layers.dense(
+                    inputs = hidden_sensor_layer,
+                    units = params['g_size'],
+                    activation = None)
+            
             hidden_location_layer = tf.layers.dense(
-                    # unsure of input here
-                    inputs = ,
+                    inputs = locs,
                     units = params['l_size'],
                     activation = tf.nn.relu)
             
+            dense_location_layer = tf.layers.dense(
+                    inputs = hidden_location_layer,
+                    units = params['l_size'],
+                    activation = None)
+            
             glimpse_out_layer = tf.layers.dense(
-                    inputs = tf.concat(
-                            values = [hidden_sensor_layer, hidden_location_layer],
-                            axis = 0),
+                    inputs = tf.add(
```

Comment:
`glimpse_out_layer` is also what should also be returned by the method.  I.e. the four lines of comments from `# NOTE:` and the `return` line following it should be replaced by `return glimpse_out_layer`